lib/transport/selenium-webdriver/browserstackAutomate.js

```diff
@@ -0,0 +1,31 @@
+const BrowserStack = require('./browserstack.js');
+const AppiumMixin = require('./appiumMixin');
+const {Capabilities} = require('selenium-webdriver');
+
+class Automate extends BrowserStack {
+  get ApiUrl() {
+    return `https://api.browserstack.com/${this.productNamespace}`;
+  }
+
+  get productNamespace() {
+    return 'automate';
+  }
+
+  createSessionOptions(argv) {
+    this.extractAppiumOptions();
```

Comment:
You're right. Since BrowserStack uses Appium for running mobile-web tests on Automate, I thought it should also accept some `appium:` configs, but from the bstack capabilities page you shared above it seems that `appium:` configs are not required on Automate.